website2/.vuepress/local-components/SponsorList.vue

```diff
@@ -50,7 +50,29 @@ export default {
   padding-right: 10px;
 }
 
+@media (max-width: 639px) {
```

Comment:
Can I recommend using `@media (min-width)` only?

This will make it align with "mobile-first" approach where are the default CSS code is for mobile and incrementally get upgraded when your screen size becomes larger.

Reply:
I have thought the same thing. 

The reason for `@media (max-width)` being here is because of those carefully crafted `img`'s`height` attributes. They already look good on desktop, hence `@media (max-width)` as a hot fix. :sweat_smile:

Reply:
Thanks for the clarification!